drinfomon.lic

```diff
@@ -852,7 +851,10 @@ while line = script.gets
         end
       end
     end
-    put('info') if DRStats.guild.nil? && !dead?
+    if DRStats.guild.nil? && !dead?
+      fput('info')
```

Comment:
using put here causes multiple 'info' commands to be sent to the game. I think we want to use fput since we want a response from the game

Reply:
No you just want to debounce the info check, don't do it more than once every 10-30 seconds.

Reply:
That sounds good